aimemalaika/Andela_Cycle_12_Chalenge_Two
# Ft tdd 169340625

## What does this PR do?
allow api to test end point using test unit and send coverage
## Description of the task to be completed
- creation of unit tests
- adding coverage to read me
## How should it be tested
- clone the repository https://github.com/aime19/Andela_Cycle_12_Chalenge_Two.git
and navigate in the path /Andela_Cycle_12_Chalenge_Two
- checkout the feature branch ft_feedback_day_one
- run npm test
```
> andela_challenge_two@1.0.0 test /Users/aimemalaika/Documents/andelaApi
> mocha --timeout 10000 --require @babel/register test/*.test.js --exit
Connected on 3000
  Test app.js
    Test welcome route
      ✓ path not found

  Test users auth
    Test signup
      ✓ user should signup
      ✓ user should signup without an email
      ✓ user should enter password twice an equal when registering
      ✓ user password should at least conatain one catpal letter, one small letter, one number and hahe a size of at least 6 character
  5 passing (63ms)
```

## Any background context to provide ?
 - N/A

## What are the relevant pivotal tracker stories
 [#169340625](https://www.pivotaltracker.com/n/projects/2401031/stories/169340625)
